### DISCUSSION TITLE
What type of proxies do i need to use?

### DISCUSSION THREAD

**Discussion by Yarumina** [3urobeat/steam-comment-service-bot#206] [Q&A] [answered]
Yo.  Do i need to use residential Proxies? or Static IP Proxies? Could you say what type you need to use it

**Comment by 3urobeat** [comment, marked as answer]
Hey, thanks for opening an issue!  
You don't need any specific type of proxies, I myself use the cheapest plan webshare.io has to offer.  
Some proxy providers may be blocked by Steam though, that's the main thing you have to watch out for.

**Comment by Yarumina** [comment]
Alright i see. Im not sure rn but User:Auth isnt required right

**Reply by 3urobeat** [reply]
No, it isn't required from the bot, but probably from your proxy provider. Otherwise everyone could use those proxies without authentication
